extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -164,15 +164,19 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
         const img: Element = node;
         const src = img.getAttribute('src') as string;
         const { mimeType, data } = this.parseInlineImageSrc(src);
-        const imgAttachment = new Attachment({
-          cid: Attachment.attachmentId(),
-          name: img.getAttribute('name') || '',
-          type: mimeType,
-          data: Buf.fromBase64Str(data),
-          inline: true
-        });
-        img.setAttribute('src', `cid:${imgAttachment.cid}`);
-        imgAttachments.push(imgAttachment);
+        if (mimeType && data) {
+          const imgAttachment = new Attachment({
+            cid: Attachment.attachmentId(),
+            name: img.getAttribute('name') || '',
+            type: mimeType,
+            data: Buf.fromBase64Str(data),
+            inline: true
+          });
+          img.setAttribute('src', `cid:${imgAttachment.cid}`);
+          imgAttachments.push(imgAttachment);
+        } else {
+          Catch.report(`Unable to parse an inline image with src="${src}"`);
```

Comment:
This reports it to us - not good - a privacy violation. You can instead do a toast to tell the user.

Reply:
Right, will do that 👍

Reply:
Instead of toast I decided to go with the error modal which was already implemented there. 

<img width="522" alt="CleanShot 2021-01-28 at 14 53 20@2x" src="https://user-images.githubusercontent.com/6059356/106141887-72ce6c80-6179-11eb-949e-f05f71928baf.png">

<img width="519" alt="CleanShot 2021-01-28 at 14 54 32@2x" src="https://user-images.githubusercontent.com/6059356/106141904-7661f380-6179-11eb-8ca2-0433e47efab4.png">

The current behavior is: we will NOT send a message if there were issues with parsing inline images, IMO it's better to prevent sending a message which is broken because of unexpected issues. @tomholub Let me know if you want some other behavior here, e.g. remove broken images and send a message without them.